library/ssl_tls13_keys.h

```diff
@@ -667,6 +667,23 @@ int mbedtls_ssl_tls13_calculate_verify_data( mbedtls_ssl_context *ssl,
                                              size_t *actual_len,
                                              int which );
 
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+/**
+ * \brief Compute TLS 1.3 early transform
+ *
+ * \param ssl  The SSL context to operate on. The early secret must have been
```

Comment:
```suggestion
 * \param ssl  The SSL context to operate on. 
```
**The early secret must have been computed** , should this be put somewhere else. such as **brief**?

Reply:
Removed. The warning in this paragraph is enough.